src/main/java/com/datatorrent/netlet/DefaultEventLoop.java

```diff
@@ -323,7 +323,12 @@ public void run()
             }
           }
           else {
-            register(channel, SelectionKey.OP_CONNECT, l);
+            /*
```

Comment:
Vlad, if OP_CONNECT is never fired, who calls ClientListener.connected()? Or by adding OP_READ, it always fires OP_CONNECT?